src/vdbfusion/pybind/vdb_volume.py

```diff
@@ -65,14 +65,6 @@ def integrate(
     ) -> None:
         ...
 
-    @overload
```

Comment:
The idea of this is just to add some type hints for text editors, I honestly hate it and I wish one can use function overloading like in C++. Did you remove it for some reason?

Reply:
I did not know the reason for having it. Isn't the complete `integrate` function sufficient for type hints? It knows the type of `points`, `extrinsic` and `weight` in case they are defined, seems to be more a limitation of the typing package you use? I might use the text editor a bit different, but I never had a need for such an overload. I am pretty sure most people are just confused by it and will never really make use of it so the advantages of having it seem not to make up the draw backs, in my opinion. Happy to put it back if you insist though.